### PR TITLE
fix(presence): Make more accessible by adding appropriate roles

### PR DESCRIPTION
**Pull request by ConradJChan**
In the `Flyout` component, added the `role="button"` to the button child element. Added `aria-hidden="true"` to the `PresenceAvatar` used in the `PresenceAvatarList` so that it is ignored by screen readers.

TODO:
- [x] update unit tests

### CODE REVIEW

File: i18n/en-US.properties [hunk 1]
```diff
@@ -1086,6 +1088,8 @@ boxui.presence.timeSinceLastCommented = Commented {timeAgo}
 boxui.presence.timeSinceLastModified = Edited {timeAgo}
 # Description for when someone last previewed a document, {timeAgo} is a relative time like 2 months ago
 boxui.presence.timeSinceLastPreviewed = Previewed {timeAgo}
+# Text on button to toggle the presence overlay
+boxui.presence.toggleOverlayLabel = Toggle Presence Overlay
```
Comment (jstoffan) [inline i18n/en-US.properties:1092]: I think the guidance is to just use the name of the element and not the action/instructions. So, just `Presence Overlay` in this case.

File: src/features/presence/PresenceAvatar.js [hunk 2]
```diff
@@ -26,6 +42,7 @@ const PresenceAvatar = ({
         onFocus={onFocus}
         onMouseEnter={onMouseEnter}
         onMouseLeave={onMouseLeave}
+        role="figure"
```
Comment (jstoffan) [inline src/features/presence/PresenceAvatar.js:45]: Is this correct for an interactive element? I think we've been removing the use of `figure` in most places, as well. It also looks like the line below has a condition of `isDropDownAvatar`.

File: src/features/presence/PresenceAvatar.js [hunk 3]
```diff
@@ -34,20 +51,4 @@ const PresenceAvatar = ({
...
-    onBlur: PropTypes.func,
-    isDropDownAvatar: PropTypes.bool,
-};
-
-PresenceAvatar.defaultProps = {
-    isActive: false,
-};
-
-export default PresenceAvatar;
+export default injectIntl(PresenceAvatar);
```
Comment (jstoffan) [inline src/features/presence/PresenceAvatar.js:54]: Do we have any cases where we're using the ref of this component? If so, do they still work properly?

File: src/features/presence/PresenceAvatarList.tsx [hunk 4]
```diff
@@ -5,6 +5,7 @@ import noop from 'lodash/noop';
 import PresenceAvatar from './PresenceAvatar';
 import PresenceAvatarTooltipContent from './PresenceAvatarTooltipContent';
 import Tooltip, { TooltipPosition } from '../../components/tooltip';
+
```
Comment (jstoffan) [inline src/features/presence/PresenceAvatarList.tsx:8]: Can we revert this whitespace-only change?

File: src/features/presence/messages.js [hunk 5]
```diff
@@ -6,6 +6,11 @@ const messages = defineMessages({
         description: 'Description for someone who is currently viewing or editing a document',
         id: 'boxui.presence.activeNow',
     },
+    avatarLabel: {
+        defaultMessage: 'Avatar for {name}',
+        description: 'Description for the label of the avatar for a given user',
```
Comment (jstoffan) [inline src/features/presence/messages.js:11]: Is this a description of the label or the text for the label?

File: src/features/presence/PresenceAvatar.js (hunk 3 above)
Comment (ConradJChan) [reply src/features/presence/PresenceAvatar.js:54]: Good callout, I poked around a few other repos and didn't find any references so we should be good here

File: i18n/en-US.properties [hunk 6]
```diff
@@ -1086,6 +1086,8 @@ boxui.presence.timeSinceLastCommented = Commented {timeAgo}
 boxui.presence.timeSinceLastModified = Edited {timeAgo}
 # Description for when someone last previewed a document, {timeAgo} is a relative time like 2 months ago
 boxui.presence.timeSinceLastPreviewed = Previewed {timeAgo}
+# Description of the button to toggle the presence overlay
+boxui.presence.toggleButtonLabel = Presence Activity
```
Comment (alexkrolick) [inline i18n/en-US.properties:1090]: Would this string make sense in context? Something like Recent Activity (the title of the list that opens up) might be more clear.

**Review by jstoffan — APPROVED** [review]

**Pull request merged**